configuration/concepts/commons_void.csv

```diff
@@ -1,33 +1,24 @@
 Uuid,Void/Retire,Fully specified name:en,_version:1,_order:50

```

Comment:
Let's merge that file and **concept_void.csv** to a **voids.csv** file

Reply:
Done